qa/tasks/cephadm.py

```diff
@@ -813,6 +813,10 @@ def ceph_bootstrap(ctx, config):
         yield
 
     finally:
+        log.info('Tearing down any agents...')
+        _shell(ctx, cluster_name, bootstrap_remote,
+                ['ceph', 'config', 'set', 'mgr', 'mgr/cephadm/use_agent', 'false'])
```

Comment:
Could cephadm do this itself from the module's `shutdown()` handler or is that non-functional?  Given that the agent is something that is spawned by cephadm module, having to deal with the tear down separately in the teuthology task isn't symmetric.

Reply:
@rkachach @adk3798 ?